src/status_im/i18n.cljs

```diff
@@ -115,9 +118,9 @@
                              :zh-hans-cn  zh-hans/translations
```

Comment:
Please organise the Chinese section here like so:
```
:zh          zh-hans/translations
                             :zh-hans     zh-hans/translations
                             :zh-hans-cn  zh-hans/translations
                             :zh-hans-mo  zh-hans/translations
                             :zh-hant     zh-hant/translations
                             :zh-hant-tw  zh-hant-tw/translations
                             :zh-hant-sg  zh-hant-sg/translations
                             :zh-hant-hk  zh-hant-hk/translations
                             :zh-hant-cn  zh-hant/translations
                             :zh-hant-mo  zh-hant/translations
                             :zh-wuu      zh-wuu/translations
                             :zh-yue      zh-yue/translations
```

Reply:
It said "we went looking everywhere, but couldn’t find those commits". So where  can I find the latest translations?

Reply:
@ninadou I use `add --amend && push -f` to make the changes into one single commit, so the hash of commit at this message may not be available now, how about go [here](https://github.com/status-im/status-react/pull/2552/files)?

Reply:
Hey @ninadou , sorry for the disturb, any updates that I should fix here?